### ISSUE TITLE
Agent crashes with many files

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kuza55/opendlp#47]
```
What steps will reproduce the problem?
1. Deploy agent to Windows 2008 R2 SP0 server with many files
2. Scan starts, OpenDLP service starts on server
3. all_dir.txt gets to about 120k and OpenDLP crashes.  Nothing in logs.

What is the expected output? What do you see instead?
Service crashes

What version of the product are you using? On what operating system?
0.4.3 in pre-configured VM

Please provide any additional information below.

I can successfully scan a subset of documents ("only scan these directories") 
on the same server.

```

Original issue reported on code.google.com by `machine...@gmail.com` on 16 Feb 2012 at 6:32

**Comment by GoogleCodeExporter** [comment]
```
FYI, the all_dir.txt file is 124,391,424 bytes and 943520 lines.
```

Original comment by `machine...@gmail.com` on 16 Feb 2012 at 7:28

**Comment by GoogleCodeExporter** [comment]
```
I have received another report of this from somebody else, too. If you are 
willing to help test a new agent binary, I can send it to you when I write a 
new directory crawling function soon.
```

Original comment by `andrew.O...@gmail.com` on 16 Feb 2012 at 7:57
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in the Windows agent shipped with OpenDLP 0.4.4, which I released 
today.

The only caveat is that the fix does not allow you to pause or uninstall the 
agent while it's doing the initial directory listing (but you can still 
pause/uninstall the agent after this directory listing is done). I will look 
into a better way of handling this in a future release, but I just wanted to at 
least get this fix out as soon as possible.
```

Original comment by `andrew.O...@gmail.com` on 22 Feb 2012 at 11:13
- Changed state: **Done**
